src/sections/Blog/Blog-grid/blogGrid.style.js

```diff
@@ -4,6 +4,9 @@ export const BlogPageWrapper = styled.div`
     .blog-grid-wrapper{
         margin: 5rem auto;
     }
+    .post-meta-block{
+        height: 3.5rem;
+        
```

Comment:
Move this under the below `media` tag for `max-width: 992px`, above that it is not required.